apiserver/facades/controller/charmrevisionupdater/testing/suite.go

```diff
@@ -92,6 +92,9 @@ func (s *CharmSuite) AddMachine(c *gc.C, machineId string, job state.MachineJob)
 	c.Assert(err, jc.ErrorIsNil)
 	inst, hc := jujutesting.AssertStartInstanceWithConstraints(c, s.jcSuite.Environ, s.jcSuite.ProviderCallContext, controllerCfg.ControllerUUID(), m.Id(), cons)
 	err = m.SetProvisioned(inst.Id(), "fake_nonce", hc)
+	instId, err := m.InstanceId()
```

Comment:
We don't need these lines I don't think. We just need to ensure the machine is provisioned hence the call to SetProvisioned()